web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerDefault.java

```diff
@@ -0,0 +1,23 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+public class LinkCheckerDefault implements LinkCheckerInterface {
+    private String getCheckUrl(Element onlineResource) {
+        return LinkCheckerUtils.parseOnlineResource(onlineResource, LinkCheckerUtils.URL_XPATH);
+    }
+
+    @Override
+    public boolean check(Element onlineResource) {
```

Comment:
This method is duplicated in each of the implementation classes.  Could be extracted to a common base class.